test/unittests/test_cpp.cpp

```diff
@@ -353,8 +353,9 @@ TEST(cpp, vm_null)
 TEST(cpp, vm_move)
 {
     static int destroy_counter = 0;
-    const auto template_vm = evmc_vm{
-        EVMC_ABI_VERSION, "", "", [](evmc_vm*) { ++destroy_counter; }, nullptr, nullptr, nullptr};
+    const auto template_vm =
+        evmc_vm{EVMC_ABI_VERSION, "",      "",     [](evmc_vm* /*vm*/) { ++destroy_counter; },
```

Comment:
I'm not a fan of this kind of indentation :)

Reply:
This is up to clang-format config. I actually was considering changing this to be aligned with other C++ projects.

Reply:
I cannot find the option responsible for it, if any exists.